source/DamageProfile.cpp

```diff
@@ -125,7 +125,7 @@ void DamageProfile::PopulateDamage(DamageDealt &damage, const Ship &ship) const
 	{
 		double piercing = max(0., min(1., weapon.Piercing() / (1. + attributes.Get("piercing protection"))
 			- attributes.Get("piercing resistance")));
-		shieldFraction = (1. - piercing) / (1. + ship.DisruptionLevel() * .01);
+		shieldFraction = (1. - min((piercing + attributes.Get("shield permeability")), 1.)) / (1. + ship.DisruptionLevel() * .01);
```

Comment:
```suggestion
		shieldFraction = (1. - min((piercing + attributes.Get("shield permeability")), 1.))
			/ (1. + ship.DisruptionLevel() * .01);
```
this should fix the style issue